src/ngraph_utils.cc

```diff
@@ -164,6 +164,17 @@ const gtl::ArraySlice<DataType>& NGraphIndexDTypes() {
   return result;
 }
 
+Status CheckAxisDimInRange(std::vector<int64> axes, size_t rank) {
+  for (auto i : axes) {
+    if (i >= -(rank) && i < rank) {
+      return errors::InvalidArgument(
+          "Axis Dimension is out of range.Should be in range [-rank(input), "
+          "rank(input))");
```

Comment:
`range.Should` -> `range. Should`

How about printing the actual numbers in the message, something like:
```
for (auto i = 0; i < axes.size(); i++) {
     if (axes[i] >= -rank && axes[i] < rank) {
       return errors::InvalidArgument(
           "Axis Dimension is out of range. Got " + axes[i] + " at position " + i +" should have be in range [-rank(input), rank(input)), that is [" + -rank + ", " + rank + ")");   //I am using string concat operator + loosely here
```